src/main/java/core/basesyntax/figure/Figure.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.figure;
+
+import core.basesyntax.supplier.Color;
+import core.basesyntax.supplier.FigureArea;
+
+public abstract class Figure implements FigureArea {
+    private Color color;
+
+    public abstract String draw();
```

Comment:
Please be consistent, if you want to define behavior (methods) in separate interfaces like you did in case of method draw and Drawable interface you should create an interface AreaCalculator with method getArea, or avoid approach to use interfaces and declare everything in abstract class. First approach is more flexible and better from my point of view

Reply:
thx, fixed